public/data/job-openings-data.json

```diff
@@ -1,5 +1,57 @@
 {
   "jobs": [
+    {
```

Comment:
@kishan-wappnet Please add your data/object at the end of file.

Reply:
object updated and set at end of file